packages/core/src/evals/run/score-persistence.test.ts

```
@@ -0,0 +1,123 @@
+      id: 'testAgent',
+      name: 'Test Agent',
+      instructions: 'Test instructions',
+      model: dummyModel,
+    });
+
+    // Create a scorer
+    const scorer = createScorer({
+      id: 'testScorer',
+      description: 'Test scorer for bug demonstration',
+      name: 'testScorer',
+    }).generateScore(({}) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)